includes/functions.php

```diff
@@ -4091,6 +4091,43 @@ function dokan_format_time( $date = '', $format = false ) {
     return dokan_format_datetime( $date, $format );
 }
 
+/**
+ * Create an expected date time format from a given format.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @param string $format      Date string format
+ * @param string $date_string Date time string
+ *
+ * @return DateTimeImmutable|false
+ */
+function dokan_create_date_from_format( $format, $date_string ) {
+    return \DateTimeImmutable::createFromFormat(
+        $format,
+        $date_string,
+        new \DateTimeZone( dokan_wp_timezone_string() )
+    );
+}
+
+/**
+ * Convert times in expected format.
+ *
+ * @param array|string $times_data    Times data
+ * @param string       $input_format  Times current format
+ * @param string       $output_format Times converted format
+ *
+ * @return array
+ */
+function dokan_get_formatted_times( $times_data = [], $input_format = 'g:i a', $output_format = 'g:i a' ) {
+    $times = [];
+    foreach ( (array) $times_data as $time ) {
+        $datetime = dokan_create_date_from_format( $input_format, $time );
+        $times[]  = $datetime->format( $output_format );
+    }
+
+    return $times;
```

Comment:
rewrite this with: `return count($times) === 1 ? $times[0] : $times`
this is necessary since we are taking input as string or array